packages/wrangler/src/__tests__/cloudchamber/apply.test.ts

```diff
@@ -1904,7 +1904,7 @@ describe("cloudchamber apply", () => {
 			│   tier = 2
 			│
 			│   [containers.configuration]
-			│   image = \\"${registry}/hello:1.0\\"
+			│   image = \\"${registry}/some-account-id/hello:1.0\\"
```

Comment:
Before, we were also omitting the namespace (account ID) in the diff that we showed to the user. This ends up being more trouble than it's worth, so we include the namespace (account ID) in the diff now. Updated this test to reflect that.